htlcswitch/link.go

```diff
@@ -2156,30 +2187,7 @@ func (l *channelLink) UpdateShortChanID() (lnwire.ShortChannelID, error) {
 		return hop.Source, err
 	}
 
-	sid := l.channel.ShortChanID()
-
-	l.log.Infof("updating to short_chan_id=%v for chan_id=%v", sid, chanID)
-
-	l.Lock()
-	l.shortChanID = sid
-	l.Unlock()
-
-	go func() {
-		err := l.cfg.UpdateContractSignals(&contractcourt.ContractSignals{
```

Comment:
Why was this removed? As is, it's used to update the contract court with the _acutal_ scid once things confirm.

Reply:
Before this PR, it wasn't possible to have a link with an unconfirmed funding tx into the switch so the code had no side effect. In this PR, I found it easier to reason about the contractcourt if it didn't change from the alias to the real, confirmed one when calling this